### PR TITLE
glib-2.58.3

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

File: Formula/ffmpeg@2.8.rb [hunk 1]
```diff
@@ -20,6 +20,7 @@ class FfmpegAT28 < Formula
 
   depends_on "freetype"
   depends_on "frei0r"
+  depends_on "jack"
```
Comment (MikeMcQuaid) [inline Formula/ffmpeg@2.8.rb]: Are you sure this is still needed after the `revision` is bumped?

Comment (tschoonj) [reply Formula/ffmpeg@2.8.rb]: This one seems to be a bit strange to me as the libavdevice dylib within ffmpeg actually has a hard compile-time and run-time dependency on `jack`. Not sure why this has been working until now.

Comment (tschoonj) [reply Formula/ffmpeg@2.8.rb]: Sure, no problem.

Comment (MikeMcQuaid) [reply Formula/ffmpeg@2.8.rb]: @tschoonj Thanks ❤️

Comment (MikeMcQuaid) [reply Formula/ffmpeg@2.8.rb]: @tschoonj Pulled, `jack` isn't necessary, can remove this.

**Comment by fxcoudert** [comment]
Thanks @tschoonj

**Pull request closed without merging**